fire_atlas_editor/lib/screens/editor_screen/widgets/toolbar.dart

```diff
@@ -1,9 +1,10 @@
 import 'package:equatable/equatable.dart';
+import 'package:fire_atlas_editor/services/storage/storage.dart';
```

Comment:
nit: relative import

Reply:
This is all messed up on the project, I will do a follow up to configure a better linter in the future when we have our flame-lint